### PR TITLE
Add Guido and Ivan as owners for typing.py

### PR DESCRIPTION
**Pull request by gvanrossum**
See https://github.com/python/cpython/pull/3966#issuecomment-349361679

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -51,3 +51,5 @@ Python/bootstrap_hash.c       @python/crypto-team
 **/*decimal*                  @rhettinger @skrah
 
 **/*idlelib*                  @terryjreedy
+
+**/*typing*                   @gvanrossum @ilevkivslyi
```
Comment (ilevkivskyi) [inline .github/CODEOWNERS]: Thanks for making the PR! There is a small typo: it should be ``@ilevkivskyi`` (the third letter from the end should be ``k``, not ``l``).

Comment (gvanrossum) [reply .github/CODEOWNERS]: Dang. And I was trying to be so careful! :-) Fixed now.

**Review by ilevkivskyi — APPROVED** [review]

**Comment by Mariatta** [comment]
Codeowners is only in `master` branch. Do we need it in the other branches?

**Comment by gvanrossum** [comment]
Let's not.

**Pull request merged**